### PR TITLE
Added "Folding" Keybindings

### PR DESCRIPTION
**Pull request by kmdalton**
I've added the following keybindings to hide and show code blocks analogous to vim folding.
* `Z, C` hides the code in the selected cell
* `Z, O` shows the code in the selected cell
* `Z, M` hides the code in all cells
* `Z, R` shows the code in all cells

I also bumped the version and added these shortcuts to `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ Shortcuts this extension introduces:
 | Ctrl-E  | Move Cells Down   |
 | Ctrl-Y  | Move Cells Up     |
 | Z, Z    | Center Cell       |
+| Z, C    | Hide Cell Code    |
+| Z, O    | Show Cell Code    |
+| Z, M    | Hide All Code     |
+| Z, R    | Show All Code     |
```
Comment (ianhi) [inline README.md:99]:
```suggestion
| Z, C    | Hide Code Cell |
| Z, O    | Show Code Cell |
| Z, M    | Hide All Code Cells |
| Z, R    | Show All Code Cells  |
```

File: History.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # History
 
+## 0.12.2 / 2020-04-25
+
+  * `z c` hides selected cell code
+  * `z o` shows selected cell code
+  * `z m` hides all cell code
+  * `z r` shows all cell code
```
Comment (ianhi) [inline History.md:8]:
```suggestion
  * `z c` hides selected code cell
  * `z o` shows selected code cell
  * `z m` hides all code cells
  * `z r` shows all code cells
```

**Comment by kmdalton** [comment]
@ianhi , I approved your changes to make the language more clear.

**Comment by ianhi** [comment]
@axelfahy thanks for adding! Does adding a new feature warrant a new release?

**Comment by axelfahy** [comment]
@ianhhi you're welcome. I am not sure what you mean by warrant, but to make the feature available easily (without cloning and compiling), a new release has to be made.

**Pull request merged**